app/javascript/packages/assets/package.json

```diff
@@ -2,6 +2,10 @@
   "name": "@18f/identity-assets",
   "private": true,
   "version": "1.0.0",
+  "exports": {
```

Comment:
Some packages are intentionally written in JavaScript because we expect them to be loaded directly by Node.js without any pre/post-processing (e.g. Webpack plugins, Sass compilation, various configurations, etc.). The way I approached this was to check for a defined entrypoint, since this would describe how Node.js would be expected to resolve a package.